### ISSUE TITLE
Characters URL-encoded twice in runInstalledQuery()

### ISSUE THREAD

**Issue by maciejmrug** [pyTigerGraph/pyTigerGraph#27]
**Describe the bug**
When passing `params` as dict into `TigerGraphConnection.runInstalledQuery()` where any of the values contains a character subject to URL-encoding, the encoding happens twice, which, for example, in the case of space character, produces `%2520` instead of expected `%20`.
1st encoding: https://github.com/pyTigerGraph/pyTigerGraph/blob/master/pyTigerGraph/pyTigerGraph.py#L1287
2nd encoding: when execution reaches `requests.models.PreparedRequest.prepare_url()`

**To Reproduce**
```python
from unittest import TestCase
from unittest.mock import patch
from requests import Session, Response

from pyTigerGraph import TigerGraphConnection


class TestPyTigerGraph(TestCase):


    @patch.object(Session, 'send')
    def test_urlencode_space(self, send):
        res = Response()
        res.status_code = 200
        res._content = b'{"results":{}}'
        send.return_value = res

        conn = TigerGraphConnection()
        conn.runInstalledQuery("whatever", params={"param": "has space"})
        actual_url = send.call_args.args[0].url

        expected_url = "http://localhost:9000/query/MyGraph/whatever?param=has%20space"
        self.assertEquals(expected_url, actual_url)
```
Results in:
<img width="1275" alt="Screenshot 2020-12-14 at 15 37 33" src="https://user-images.githubusercontent.com/14819187/102094203-57560f00-3e22-11eb-8230-e84eb2ebd056.png">

**Expected behavior**
The URL-escaping for the params behave correctly

**Environment**
```
Python 3.8.6 (default, Oct 27 2020, 08:57:44) 
[Clang 12.0.0 (clang-1200.0.32.21)] on darwin
```

**Comment by zrougamed** [comment]
hey @maciejmrug 

thanks for the bug report please note that this issue is fixed now the version is v0.0.9.6.3

please do :

pip install pyTigerGraph --upgrade

Thanks!

**Comment by maciejmrug** [comment]
@zrougamed Thank you for a quick response and fix! :)
